packages/engine/src/scene/components/VolumetricComponent.ts

```diff
@@ -44,6 +45,9 @@ export const VolumetricComponent = defineComponent({
   onSet: (entity, component, json) => {
     if (typeof json?.useLoadingEffect === 'boolean' && json.useLoadingEffect !== component.useLoadingEffect.value)
       component.useLoadingEffect.set(json.useLoadingEffect)
+    if (!hasComponent(entity, MediaComponent)) {
```

Comment:
the hasComponent check is redundant here, setComponent can be called multiple time

Reply:
Removed it